.github/workflows/scripts/only_latest_easystacks.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
I'd add a short header (see other .sh scripts) with:
- purpose of the script
- author(s)
- license

Reply:
Done in https://github.com/EESSI/software-layer/pull/795/commits/3ec0157ef11001586de4eb72c0e3e7bbd5ac6732